lib/led/rgb.js

```diff
@@ -182,7 +182,8 @@ function RGB(opts) {
     blue: 255,
     intensity: 100,
     isAnode: opts.isAnode || false,
-    interval: null
+    interval: null,
+    isRunning: false
```

Comment:
`isRunning` is used to track whether an `Animation` (requiring more detail that `setInterval`) is in progress.  Copying the pattern used in led.js here.